orion/controllers/semesters/semestersModel.go

```diff
@@ -0,0 +1,14 @@
+package semesters
```

Comment:
nit: `semesterModel.go`

Reply:
Why is the package also semestersModel.go